StudCity.Frontend/src/components/Sidenav/FullNavbar.jsx

```diff
@@ -1,14 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Switch from '../Switch';
 import MenuComponents from './MenuItems';
+import Button from '../../UI/Button';
+import CustomDialog from '../../UI/CustomDialog'
+import CreateChatModal from './CreateChatModal';
 
 const FullNavbar = ({ active, activeName }) => {
+	const [isOpenCreateChatModal, setIsOpenCreateChatModal] = useState(false);
+
+	const handleClick = () => {
```

Comment:
probably better name 'handleOpenModel' but can be as it is

Reply:
Yeap agree with @Serhiy-dotcom rename pls)

Reply:
@Vova-Romaniuk I was talking about function name, not the state